subprojects/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/logging/comparison/ExhaustiveLinesSearcher.java

```diff
@@ -0,0 +1,183 @@
+/*
+ * Copyright 2022 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.integtests.fixtures.logging.comparison;
+
+import org.apache.commons.lang3.StringUtils;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.Optional;
+import java.util.Set;
+
+public class ExhaustiveLinesSearcher {
+    private boolean matchBlankLines = false;
+    private boolean showLessLikelyMatches = false;
+    private boolean useUnifiedDiff = false;
+
+    /**
+     * Instructs empty lines to be considered matches when exhaustively building the list of potential matches.
+     * <p>
+     * Enabling this option could really explode the number of potential matches for common output cases
+     * where blank lines exist in both the expected and actual lines.
+     *
+     * @return {@code this)
+     */
+    public ExhaustiveLinesSearcher matchesBlankLines() {
```

Comment:
Make this consistent with the other method names:
```suggestion
    public ExhaustiveLinesSearcher matchBlankLines() {
```